reltime/reltime.py

```diff
@@ -164,16 +164,29 @@ def split_am_pm(time):
     return time, am_pm
 
 
+def last_day_of_month(date_data):
```

Comment:
I don't really grok how this works, seems a bit safer to use the built in python tools? https://docs.python.org/3/library/calendar.html#calendar.monthrange